purpleair-aqi.js

```diff
@@ -1,32 +1,71 @@
-// This code is from <https://github.com/jasonsnell/PurpleAir-AQI-Scriptable-Widget>
-// By Jason Snell based on code by Matt Silverlock, Rob Silverii, Adam Lickel
+"use strict";
```

Comment:
Adding this at the top of the script prevents mistakes like accessing variables without declaring them.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode